stubs/Modules/Base/App/Console/Commands/GenerateSitemap.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace RemiHin\FilamentPrefabStubs\Modules\Base\App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Str;
+use Spatie\Sitemap\SitemapIndex;
+use function App\Console\Commands\public_path;
+
+class GenerateSitemap extends Command
+{
+    protected $signature = 'app:generate-sitemap';
+
+    protected $description = 'Generate a fresh sitemap.xml';
+
+    protected $chunkSize = 100;
+
+    public function handle(): int
+    {
+        // Delete old sitemaps
+        foreach (scandir(public_path()) as $file) {
+            if (Str::of($file)->contains('sitemap')) {
+                unlink(public_path('/') . $file);
+            }
+        }
+
+        // Create new sitemap.xml index file
+        $sitemapIndex = SitemapIndex::create();
+
+        $models = collect([
+            // Add classes with pages here "Model::class,"
```

Comment:
Dit zou een mooie zijn om in een config file op te slaan.

En voeg dan aan de Readme even toe, hoe dit te gebruiken. Zodat je daar beschrijft welke config aangepast moet worden om sitemap te genereren